.github/workflows/WebAssembly.yml

```diff
@@ -54,6 +54,9 @@ jobs:
         git clone --recurse-submodules https://github.com/duckdb/duckdb-wasm
         cd duckdb-wasm
         git checkout ${{ env.DUCKDB_WASM_VERSION }}
+        for filename in ../.github/patches/duckdb-wasm/*.patch; do
```

Comment:
How are you going to make sure the patches are applied in the right order if there's more than one?